### PR TITLE
BaSP-MR-186: Component Cards - Activity in Members

### PR DESCRIPTION
**Pull request by MaateoC**
I added Cards that demostrate the activities which should be created within the Members tab

### CODE REVIEW

**Review by CristianLotorto — APPROVED** [review]
Looks great Mate! nice work!

**Review by IvanJukonis — CHANGES_REQUESTED** [review]
Changes on this component are being on discussion.

File: src/Components/Users/Member/activities/Cards/cards.module.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+.card a {
+  display: block;
+  margin: 20px 40px;
+  margin-top: 50px;
+  background-color: #AACE9B;;
+  color: #373867;
+  padding: 15px 20px;
+  text-align: center;
+  text-decoration: none;
```
Comment (YhanahiRosenthal) [inline src/Components/Users/Member/activities/Cards/cards.module.css:39]: here it has two ";" and next lines i think no work

**Review by YhanahiRosenthal — CHANGES_REQUESTED** [review]
i commented something!

**Review by IvanJukonis — APPROVED** [review]

**Review by YhanahiRosenthal — APPROVED** [review]
Good job!

**Pull request merged**